packages/components/ovh-at-internet/src/utils.ts

```diff
@@ -1,6 +1,44 @@
 export function debug(...args: any[]) {
-  if (!window.test && //temporarily  fix jest bug
-    window.localStorage?.getItem('MANAGER_TRACKING_DEBUG')) {
+  if (
+    !window.test && // temporarily  fix jest bug
+    window.localStorage?.getItem('MANAGER_TRACKING_DEBUG')
+  ) {
     console.debug(...args);
   }
 }
+
+export const isEmpty = (obj: Record<string, unknown>) => {
+  if (obj?.length || obj?.size) return false;
+  if (typeof obj !== 'object') return true;
+  return !Object.values(obj).length;
+};
```

Comment:
I think we should specialize this function for each type that is used because this function is too generic

Reply:
it is only using on object in this project, actually the type of `obj` should be `unknown`

Reply:
I think we should keep a function for each : isSetEmpty, isObjectEmpty... because if we can only have a map we wouldn't recreate a function and it will be worst to refactor later

Reply:
I get your point and you are right, but to me it's kinda out of scope, the goal of this PR is to remove `lodash` so I want to just replace the functions used from `lodash` to avoid any breaking changes, unexpected side effects. If we want to improve the code of this package I'd suggest refactoring it in another PR

Reply:
What is the point to remove lodash if we copy the functions inside the project ? To me we need to remove lodash to avoid using functions that already exists in native javascript

Or maybe it is to reduce bundle size ? But isn't lodash tree-shaked ?

Reply:
it is indeed to reduce bundle size, and while `lodash` supports tree-shaking, it is not used in this project as it imports the full library

so instead of just moving to treeshaked dependency declarations I made the decision to replace the function instead to remove any dependence to this package (I think we should aim to this goal for the whole manager project)